doc/specs/tagging_technical.md

```diff
@@ -0,0 +1,116 @@
+# Tagging - Technical Specifications
+
+## Preface
+
+The tagging system is part of the core of meteor-gazelle. Tagging allows users to add labels
+to their items, simplifying the way different sections of the site are searched. It allows
+for more fine-graned queries, limiting results to the desired item set.
+
+## Class Implementation
+
+* Class: TagManager - Attached to individual items (threads, wiki articles, etc.)
```

Comment:
Can we just like, write the method signatures and docstrings or whatever JS calls them, in code blocks? This is pretty impossible to read, doesn't give argument names, etc